magento/pwa-studio
# [PWA-837] Localize CMS

## Description

Localize all strings within the CMS area of the application (CMS Pages, Home Page). Include localized strings in default en_US language pack

## Related Issue





https://jira.corp.magento.com/browse/PWA-837

## Acceptance 



### Verification Stakeholders

@dpatil-magento 
@sirugh 
### Specification



### Verification Steps


1. Go to Home Page and CMS Pages
2. Verify strings are translated where applicable

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
